dist/drawer-dialog/ds6/drawer-dialog.css

```diff
@@ -70,6 +70,16 @@
 .drawer-dialog__main > :last-child {
   margin-bottom: 0;
 }
+.drawer-dialog__footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+  position: relative;
+}
+.drawer-dialog__footer > :not(:first-child) {
+  margin-top: 16px;
```

Comment:
Seeing a discrepancy between dist and src here. Maybe need to regenerate dist files?

Reply:
Hmm might be a storybook thing. I thought it regenerated files automatically when you're editing. 
Either way, it's fixed now.